pkg/pillar/types/vaultmgrtypes.go

```diff
@@ -19,3 +19,27 @@ type VaultStatus struct {
 func (status VaultStatus) Key() string {
 	return status.Name
 }
+
+//VaultKeyFromDevice is published by vaultmgr towards Controller (through zedagent)
+type VaultKeyFromDevice struct {
+	Name     string
+	VaultKey []byte
```

Comment:
Can we name this EncryptedVaultKey across the board?

Reply:
Done